webapp/crashstats/supersearch/tests/test_views.py

```diff
@@ -15,31 +16,17 @@
 from django.utils.encoding import smart_str
 
 from crashstats.crashstats.models import BugAssociation
-from crashstats.crashstats.tests.test_views import BaseTestViews
-from crashstats.supersearch.models import Query, SuperSearchUnredacted
-from socorro.lib import BadArgumentError
-
-
-class TestViews(BaseTestViews):
-    def setUp(self):
-        super().setUp()
-        # Mock get_versions_for_product() so it doesn't hit supersearch breaking the
-        # supersearch mocking
-        self.mock_gvfp = mock.patch(
-            "crashstats.crashstats.utils.get_versions_for_product"
-        )
-        self.mock_gvfp.return_value = ["20.0", "19.1", "19.0", "18.0"]
```

Comment:
I took out the mock for featured versions. Because of that, we now have to index a sufficient amount of crash report data to coerce the featured version code to return the values we're looking for.

Reply:
Also, we have to use real supported products--we can't use Waterwolf and other fake products.